inception-project/inception
# #4102 - Typo in feature option

**What's in the PR**
- Fix typo by overhauling the feature details a bit and updating the documentation accordingly

**How to test manually**
* Check feature details editor
* Check documentation

**Automatic testing**

**Documentation**
